reference/docs-conceptual/learn/application-control.md

```diff
@@ -0,0 +1,51 @@
+---
+description: You can secure your PowerShell environments on Windows using Windows Defender Application Control (WDAC).
+ms.date: 05/23/2023
+title: Using Windows Defender Application Control
+---
+# Using Windows Defender Application Control
+
+Windows 10 includes two technologies, [Windows Defender Application Control (WDAC)][04] and
+[AppLocker][01] that can be used for application control. They allow you to create a lockdown
+experience to meet your organization's specific scenarios and requirements.
+
+> [!NOTE]
+> When it comes to [choosing between WDAC or AppLocker][03] it's generally recommended that
+> customers implement application control using WDAC rather than AppLocker. WDAC is undergoing
+> continual improvements and will be getting added support from Microsoft management platforms.
+> Although AppLocker will continue to receive security fixes, it will not receive feature
+> enhancements.
+
+**WDAC** was introduced with Windows 10 and allows organizations to control which drivers and
+applications are allowed to run on their Windows 10 devices. WDAC is designed as a security feature
+under the servicing criteria defined by the Microsoft Security Response Center (MSRC).
+
+**AppLocker** builds on the application control features of Software Restriction Policies. AppLocker
+contains new capabilities and extensions that enable you to create rules to allow or deny apps from
+running based on unique identities of files and to specify which users or groups can run those apps.
+
+For more information about AppLocker and Windows Defender Application Control (WDAC), see
+[Application Controls for Windows][04] and [WDAC and AppLocker feature availability][02].
+
+## WDAC policy enforcement
+
+When PowerShell runs under a Windows Defender Application Control (WDAC) policy, it changes its
```

Comment:
Do we need to redefine WDAC in each section, or is the definition at the top of the article enough?

Reply:
Once is enough.